sipi/scripts/sipi.init.lua

```diff
@@ -72,36 +76,39 @@ function pre_flight(prefix, identifier, cookie)
     if webapi_port == nil then
         webapi_port = config.knora_port
     end
-    server.log("webapi_port: " .. webapi_port, server.loglevel.LOG_DEBUG)
+    server.log(os.date("!%Y-%m-%dT%H:%M:%S") .. " - pre_flight - webapi_port: " .. webapi_port, server.loglevel.LOG_DEBUG)
 
     knora_url = 'http://' .. webapi_hostname .. ':' .. webapi_port .. '/admin/files/' .. prefix .. '/' .. identifier
 
-    -- print("knora_url: " .. knora_url)
-    server.log("pre_flight - knora_url: " .. knora_url, server.loglevel.LOG_DEBUG)
+    server.log(os.date("!%Y-%m-%dT%H:%M:%S") .. " - pre_flight - knora_url: " .. knora_url, server.loglevel.LOG_DEBUG)
 
     success, result = server.http("GET", knora_url, knora_cookie_header, 5000)
 
     -- check HTTP request was successful
     if not success then
-        server.log("Server.http() failed: " .. result, server.loglevel.LOG_ERR)
+        server.log(os.date("!%Y-%m-%dT%H:%M:%S") .. " - pre_flight - Server.http() failed: " .. result,
+            server.loglevel.LOG_ERR)
         return 'deny'
     end
 
     if result.status_code ~= 200 then
-        server.log("Knora returned HTTP status code " .. result.status_code, server.loglevel.LOG_ERR)
-        server.log(result.body, server.loglevel.LOG_ERR)
+        server.log(os.date("!%Y-%m-%dT%H:%M:%S") .. " - pre_flight - Knora returned HTTP status code " ..
```

Comment:
I know it is out of scope, but would it be possible to have a follow-up task to clean-up "knora"/"Knora" in this file and replace it with "dsp"?

Reply:
you're right, I'll do it